tsl/test/sql/remote_txn.sql

```diff
@@ -116,8 +138,10 @@ FROM _timescaledb_internal.show_connection_cache() ORDER BY 1,4;
 --and be rolled back with no unresolved state
 BEGIN;
     SELECT remote_node_killer_set_event('pre-commit', 'loopback');
+    SELECT debug_waitpoint_enable('remote_conn_xact_end');
```

Comment:
Can we make enabling the waitpoint part of `remote_node_killer_set_event`? These two functions are always paired together.

Similarly, can the node killer release the waitpoint automatically after it completes? Then we wouldn't have to add these extra calls.

Reply:
The waitpoint has to be released only after transaction completion. We cannot release it earlier because then we won't be able to make the remote backend wait inside the committransaction code paths as desired.

Reply:
Isn't `XACT_EVENT_COMMIT` "after" (or during) commit? So, it would be a good place to also release the waitpoint?

Reply:
The idea is for the remote backend to wait deep down in the committransaction.